cloudify_cli/commands/snapshots.py

```diff
@@ -62,10 +64,23 @@ def restore(snapshot_id,
         recreate_deployments_envs,
         force,
         tenant_name,
-        restore_certificates
+        restore_certificates,
+        no_reboot
     )
     logger.info("Started workflow execution. The execution's id is {0}".format(
         execution.id))
+    if not restore_certificates:
+        return
+    if no_reboot:
+        logger.warn('Certificates may be replaced during snapshot restore, '
```

Comment:
Certificates ~may~ **might** be replaced during **a** snapshot restore **action.** ~,~